pos_product_display_default_code/static/src/js/models.js

```diff
@@ -0,0 +1,14 @@
+odoo.define("pos_product_display_default_code.models", function (require) {
+    "use strict";
+    var models = require("point_of_sale.models");
+
+    models.PosModel.prototype.models.some(function (model) {
+        if (model.model !== "product.product") {
+            return false;
+        }
+        model.context = function () {
```

Comment:
Question : You should maybe add a key to the context, instead of overwrite. Otherwise other modules will fail. Dont you think ?

Reply:
Thanks !